tsconfig.json

```diff
@@ -19,6 +19,6 @@
   "buildOnSave": false,
   "angularCompilerOptions": {
     "skipTemplateCodegen": true,
-    "enableIvy": false
+    "enableIvy": true
```

Comment:
This is discouraged by Angular CLI itself as it makes librairies not compatible with previous versions, it is totally fine to leave `false`